toolkit/tools/imagecustomizer/container/imagecustomizer.Dockerfile

```diff
@@ -7,6 +7,14 @@ FROM ${BASE_IMAGE}
 RUN tdnf update -y && \
    tdnf install -y qemu-img rpm coreutils util-linux systemd openssl \
       sed createrepo_c squashfs-tools cdrkit parted e2fsprogs dosfstools \
-      xfsprogs zstd veritysetup grub2 grub2-pc systemd-ukify binutils lsof
+      xfsprogs zstd veritysetup grub2 grub2-pc systemd-ukify binutils lsof \
+      python3 python3-pip && \
+   tdnf clean all
 
 COPY . /
+
+# Create virtual environment and install Python dependencies for telemetry
+RUN python3 -m venv /opt/telemetry-venv && \
+   /opt/telemetry-venv/bin/pip install --no-cache-dir -r /usr/local/bin/requirements.txt
+
+ENTRYPOINT ["/usr/local/bin/entrypoint.sh"]
```

Comment:
Remove `ENTRYPOINT` for now. In places where we enable this, we can use `--entrypoint` in the `docker` command.